Doc/c-api/frame.rst

```diff
@@ -132,14 +132,39 @@ See also :ref:`Reflection <reflection>`.
    .. versionadded:: 3.11
 
    .. versionchanged:: 3.13
-      As part of :pep:`667`, return a proxy object for optimized scopes.
+      As part of :pep:`667`, return an instance of :c:var:`PyFrameLocalsProxy_Type`.
 
 
 .. c:function:: int PyFrame_GetLineNumber(PyFrameObject *frame)
 
    Return the line number that *frame* is currently executing.
 
 
+Frame Locals Proxies
+^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 3.13
+
+In older versions of :term:`CPython`, the exposed local variables dictionary
+(via :attr:`~frame.f_locals`, :c:func:`PyFrame_GetLocals`, or :func:`locals`)
```

Comment:
```suggestion
(via :attr:`~frame.f_locals`, :c:func:`PyFrame_GetLocals` or :func:`locals`)
```

Reply:
We allow the Oxford comma in general, so it's fine.

Reply:
This is an Oxford comma. I'm not sure what the precedent for us is, I'll double check.